README.rst

```diff
@@ -39,29 +39,25 @@ the hca package by running `make install` in the repository root directory.
 
 To use the command line interface with a local or test DSS, first run ``hca`` (or ``scripts/hca`` if you want to use the
 package in place from the repository root directory). This will create the file ``~/.config/hca/config.json``, which you
-can modify to update the value of `DSSClient.swagger_url` to point to the URL of the Swagger definition served by your
-DSS deployment. Lastly, the CLI enforces HTTPS connection to the DSS API. If you are connecting to a local DSS, make this
-change in ``dcp-cli/hca/util/__init__.py`` in the `SwaggerClient` object::
+can modify to update the value of ``DSSClient.swagger_url`` to point to the URL of the Swagger definition served by your
+DSS deployment. Lastly, the CLI enforces HTTPS connection to the DSS API. If you are connecting to a local DSS, make
+this change in ``dcp-cli/hca/util/__init__.py`` in the ``SwaggerClient`` object::
 
     scheme = "http"
 
+To use the Python interface with a local or test DSS, pass the URL of the Swagger definition to the ``DSSClient``
+constructor via the ``swagger_url`` parameter::
+
+    client = DSSClient(swagger_url="https://dss.example.com/v1/swagger.json")
+    res = client.post_search(...)
```

Comment:
nit: I'd just leave out the call to `post_search`. It just demonstrates unrelated usage of the API, no?